src/sentry/api/serializers/models/projectownership.py

```diff
@@ -1,7 +1,21 @@
+from typing_extensions import TypedDict
+
 from sentry.api.serializers import Serializer, register
 from sentry.models.projectownership import ProjectOwnership
 
 
+# JSON object representing this serializer in API response
+class ProjectOwnershipResponse(TypedDict):
+    raw: str
+    fallthrough: bool
+    dateCreated: int
+    lastUpdated: int
+    isActive: bool
+    autoAssignment: str
+    codeownersAutoSync: bool
+    schema: dict
```

Comment:
It looks like the schema key isn't always set, so we need to move into another TypedDict where the fields are set as optional using `total=False`. Also timestamps must be type datetime or the apidocs test will complain
```suggestion
class ProjectOwnershipResponseOptional(TypedDict, total=False):
    schema: dict
    
from datetime import datetime

class ProjectOwnershipResponse(ProjectOwnershipResponseOptional):
    raw: str
    fallthrough: bool
    dateCreated: datetime
    lastUpdated: datetime
    isActive: bool
    autoAssignment: str
    codeownersAutoSync: bool
```